include/librealsense2/dds/topics/control/control-msg.h

```diff
@@ -0,0 +1,98 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+#include <string>
+#include <librealsense2/h/rs_sensor.h>
+#include "controlPubSubTypes.h"
+
+namespace librealsense {
+namespace dds {
+namespace topics {
+namespace device {
+class control
+{
+public:
+    using type = raw::device::controlPubSubType;
+
+    //Open uses a vector of profiles, to have constant message size we limit it
+    static const size_t MAX_OPEN_PROFILES = 8;
+
+#pragma pack( push, 1 )
+    enum class control_type : uint16_t
+    {
+        SENSOR_OPEN,
+        SENSOR_CLOSE,
+        MAX_CONTROL_TYPE
+    };
+
+    struct stream_profile
+    {
+        int16_t framerate;       // FPS
+        rs2_format format;       // Transfer as uint8_t?
+        rs2_stream type;         // Transfer as uint8_t?
+        int16_t width;           // Resolution width [pixels]
+        int16_t height;          // Resolution width [pixels]
+    };
+
+    struct sensor_open_msg
+    {
+        uint32_t message_id; // Running counter
+        int16_t sensor_uid;
```

Comment:
Please add comment, UID we provide for DDS? librealsense UID?

Reply:
Added comment "Sensor ID as received in notification::sensor_header_msg.index"

This is currently the UID that was received from the USB in the server

Reply:
Done